catkin_tools/resultspace.py

```diff
@@ -79,7 +79,7 @@ def get_resultspace_environment(result_space_path, quiet=False, cached=True):
         return {}
 
     # Construct a command list which sources the setup file and prints the env to stdout
-    norc_flags = {'bash': '--norc', 'zsh': '-f'}
+    norc_flags = {'bash': '--norc', 'sh': '', 'zsh': '-f'}
```

Comment:
Maybe we should stick with `/bin/bash` as a default then, as initially suggested. If the need for other shells arises, they can be explicitly added? Otherwise this seems like fishing in the dark without a use case.